apps/wallet-mobile/src/features/ReviewTx/useCases/ReviewTxScreen/Overview/OverviewTab.tsx

```diff
@@ -0,0 +1,355 @@
+import {useSelectedWallet} from '../../../../WalletManager/common/hooks/useSelectedWallet'
+import {useWalletManager} from '../../../../WalletManager/context/WalletManagerProvider'
+import {Address} from '../../../common/Address'
+import {CollapsibleSection} from '../../../common/CollapsibleSection'
+import {Divider} from '../../../common/Divider'
+import {useAddressType} from '../../../common/hooks/useAddressType'
+import {useStrings} from '../../../common/hooks/useStrings'
+import {TokenItem} from '../../../common/TokenItem'
+import {FormattedOutputs, FormattedTx} from '../../../common/types'
+
+export const OverviewTab = ({tx}: {tx: FormattedTx}) => {
+  const {styles} = useStyles()
+
+  const notOwnedOutputs = React.useMemo(() => tx.outputs.filter((output) => !output.ownAddress), [tx.outputs])
+  const ownedOutputs = React.useMemo(() => tx.outputs.filter((output) => output.ownAddress), [tx.outputs])
+
+  return (
+    <View style={styles.root}>
+      <Space height="lg" />
+
+      <WalletInfoSection tx={tx} />
+
+      <Divider verticalSpace="lg" />
+
+      <SenderSection tx={tx} notOwnedOutputs={notOwnedOutputs} ownedOutputs={ownedOutputs} />
+    </View>
+  )
+}
+
+const WalletInfoSection = ({tx}: {tx: FormattedTx}) => {
+  const {styles} = useStyles()
+  const strings = useStrings()
+  const {wallet, meta} = useSelectedWallet()
+  const {walletManager} = useWalletManager()
+  const {plate, seed} = walletManager.checksum(wallet.publicKeyHex)
+  const seedImage = new Blockies({seed}).asBase64()
+
+  return (
+    <>
+      <View style={styles.infoItem}>
+        <Text style={styles.infoLabel}>{strings.walletLabel}</Text>
+
+        <View style={styles.plate}>
+          <Icon.WalletAvatar image={seedImage} style={styles.walletChecksum} size={24} />
+
+          <Space width="xs" />
+
+          <Text style={styles.walletInfoText}>{`${plate} | ${meta.name}`}</Text>
+        </View>
+      </View>
+
+      <Space height="sm" />
+
+      <FeeInfoItem fee={tx.fee.label} />
+    </>
+  )
+}
+
+const FeeInfoItem = ({fee}: {fee: string}) => {
+  const {styles} = useStyles()
+  const strings = useStrings()
+
+  return (
+    <View style={styles.infoItem}>
+      <Text style={styles.infoLabel}>{strings.feeLabel}</Text>
+
+      <Text style={styles.fee}>{fee}</Text>
+    </View>
+  )
+}
+
+const SenderSection = ({
+  tx,
+  notOwnedOutputs,
+  ownedOutputs,
+}: {
+  tx: FormattedTx
+  notOwnedOutputs: FormattedOutputs
+  ownedOutputs: FormattedOutputs
+}) => {
+  const strings = useStrings()
+  const address = ownedOutputs[0]?.rewardAddress ?? ownedOutputs[0]?.address
+
+  return (
+    <CollapsibleSection label={strings.myWalletLabel}>
+      <Space height="lg" />
+
+      <Address address={address} />
+
+      <Space height="sm" />
+
+      <SenderTokens tx={tx} notOwnedOutputs={notOwnedOutputs} />
+
+      {notOwnedOutputs.length === 1 && <ReceiverSection notOwnedOutputs={notOwnedOutputs} />}
```

Comment:
Is it intended when sending assets to yourself (maybe to reorganize utxos) to not show a receiving side?
![image](https://github.com/user-attachments/assets/a372b12c-e0eb-44c3-81a2-d440c48597da)

Reply:
its specified in this way https://figma.com/design/KeBVo6rWNYFLXwTphz9lHi/Mobile-App?node-id=26010-28671&node-type=shape_with_text&m=dev

In my personal opinion it makes sense because you are just paying the fee and it is reflected on that screen.

Good point anyway.